doc/src/manual/types.md

```diff
@@ -1374,7 +1374,7 @@ for cases where you don't need a more elaborate hierarchy.
 julia> struct Val{x}
        end
 
-julia> Base.@pure Val(x) = Val{x}()
+julia> Base.@unsafe_pure Val(x) = Val{x}()
```

Comment:
The `@unsafe_pure` annotation should be unnecessary (`@inline` should achieve the same results, but should also be unnecessary since this function is small)

Reply:
Removing the annotation has the added benefit of getting `@[unsafe_]pure` out of the manual.